.mailmap

```diff
@@ -1497,6 +1498,7 @@ shruti <shrutishrm512@gmail.com>
 shubhayu09 <guptashubhayu601@gmail.com> shubhayu09 <87441886+shubhayu09@users.noreply.github.com>
 sirnicolaf <43586954+sirnicolaf@users.noreply.github.com>
 slacker404 <pchantza@gmail.com>
+sonntagsgesicht <sonntagsgesicht@icloud.com>
```

Comment:
Is this a different person from Jan-Philipp Hoffman?

Adding a separate entry like this implies that it is and that two separate names should be added to the AUTHORS file.

Reply:
No,  _sonntagsgesicht_ is not a different person from _Jan-Philipp Hoffmann_. But since there are commits form both accounts, the tests required both mentioned in the AUTHORS file. 
If there is any way to drop _sonntagsgesicht_, it would be fine. But I don't know how.

Reply:
One way would be to squash the commits and add a single commit under one author name.

On the other hand if sonntagsgesicht is a different person and did author part of this work then I would have thought that they should be credited although they might prefer for their real name to be listed in the AUTHORS file. If that is the case then instead of the above a line can be added like:
```
Real Name <preferred email address> sonntagsgesicht <sonntagsgesicht@icloud.com>
```
That tells `git` and the `mailmap_check` script that commits with the latter metadata should be attributed to Real Name and ensures that Real Name will be listed in the AUTHORS file.

Reply:
Sorry, I missed the word "not" above.

Reply:
As I'm not familiar to squash commits and really afraid to ruin the whole pull request, so I preferred to fix the `.mailmap` file.  Therefore I changed my entries to

```
Jan-Philipp Hoffmann <sonntagsgesicht@icloud.com> Jan-Philipp Hoffmann <90828785+jan-philipp-hoffmann@users.noreply.github.com>
Jan-Philipp Hoffmann <sonntagsgesicht@icloud.com> sonntagsgesicht <sonntagsgesicht@icloud.com>
```

Sorry, if this doesn't work properly.  Please help me, how to fix this. I'm a bit lost.

Reply:
I think that the .mailmap file is correct now.